.travis.yml

```diff
@@ -1,9 +1,18 @@
 language: python
-python:
-  - '3.4'
-  - '2.7'
+os: linux
 
 before_install:
   - pip install flake8
-script:
-  - flake8 pynetgear
+
+jobs:
+  include:
+    - stage: lint
+      python: 3.7
+      script: flake8 pynetgear
+    - stage: test
+      python: 3.7
+      script: python test/test_getAttachedDevices.py
+
+stages:
+  - lint
+  - test
```

Comment:
I think Gitlab CI + Travis should be modified in a new PR, also #79 is taking care of Travis